### PR TITLE
restore: add faq for rule not found

### PR DESCRIPTION
**Pull request by RobertCheng-956**
### What is changed, added or deleted? (Required)

Translate docs cn pr#14383
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/14383
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: faq/backup-and-restore-faq.md [hunk 1]
```diff
@@ -296,6 +296,10 @@ Note that even if you configures [table filter](/table-filter.md#syntax), **BR d
 - System variable tables (`mysql.tidb`, `mysql.global_variables`)
 - [Other system tables](https://github.com/pingcap/tidb/blob/master/br/pkg/restore/systable_restore.go#L31)
 
+### How to deal with the error of `cannot find rewrite rule` when at restoration?
```
Comment (dveeden) [inline faq/backup-and-restore-faq.md:299]:
```suggestion
### How to deal with the error of `cannot find rewrite rule` during restoration?
```

File: faq/backup-and-restore-faq.md [hunk 2]
```diff
@@ -296,6 +296,10 @@ Note that even if you configures [table filter](/table-filter.md#syntax), **BR d
...
+
+Please examine whether there are tables in the recovery cluster sharing the same name as other tables in the backup data but having inconsistent structures. This issue often arises due to missing indexes in the tables of the recovery cluster. A recommended approach is to delete such tables in the recovery cluster first and then proceed with the restoration.
```
Comment (ran-huang) [inline faq/backup-and-restore-faq.md:301]:
```suggestion
Examine whether there are tables in the restoration cluster sharing the same name as other tables in the backup data but having inconsistent structures. In most cases, this issue is caused by missing indexes in the tables of the restoration cluster. A recommended approach is to delete such tables in the restoration cluster first and then retry restoration.
```

**Review by ran-huang — COMMENTED** [review]
rest LGTM

**Review by dveeden — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**